### PR TITLE
[docs] Changelog entries for symbol-placement: line-center

### PR DESCRIPTION
**Pull request by ChrisLoer**
Oops, forgot to include these with #12337.

/cc @fabian-guerra @tobrun @ansis

### CODE REVIEW

File: platform/ios/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed a crash that occurred when the user started a gesture before the drift animation for a previous gesture was complete. ([#12148](https://github.com/mapbox/mapbox-gl-native/pull/12148))
 * Token string syntax (`"{token}"`) in `MGLSymbolStyleLayer` `text` and `iconImageName` properties is now correctly converted to the appropriate `NSExpression` equivalent. ([#11659](https://github.com/mapbox/mapbox-gl-native/issues/11659))
 * Added an `MGLMapView.locationManager` property and `MGLLocationManager` protocol for tracking user location using a custom alternative to `CLLocationManager`. ([#12013](https://github.com/mapbox/mapbox-gl-native/pull/12013))
+* Add support for rendering `symbol-placement: line-center`. Add corresponding `MGLSymbolPlacementLineCenter` option to `MGLSymbolStyleLayer.symbolPlacement`. ([#12337](https://github.com/mapbox/mapbox-gl-native/pull/12337))
```
Comment (fabian-guerra) [inline platform/ios/CHANGELOG.md]: For changelogs, when we use verbs at the beginning of a sentence we prefer the past tense form. What do you think about?:
> Added a new option to `MGLSymbolPlacement`, `MGLSymbolPlacementLineCenter`, that places the label relatively to the center of the geometry. ([#12337](https://github.com/mapbox/mapbox-gl-native/pull/12337))

Comment (ChrisLoer) [reply platform/ios/CHANGELOG.md]: Sure, updated.

File: platform/macos/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ## Styles and rendering
 
 * Token string syntax (`"{token}"`) in `MGLSymbolStyleLayer` `text` and `iconImageName` properties is now correctly converted to the appropriate `NSExpression` equivalent. ([#11659](https://github.com/mapbox/mapbox-gl-native/issues/11659))
+* * Added a new option to `MGLSymbolPlacement`, `MGLSymbolPlacementLineCenter`, that places the label relative to the center of the geometry. ([#12337](https://github.com/mapbox/mapbox-gl-native/pull/12337)) ([#12337](https://github.com/mapbox/mapbox-gl-native/pull/12337))
```
Comment (friedbunny) [inline platform/macos/CHANGELOG.md]: This accidentally links twice to the same issue.

**Comment by friedbunny** [comment]
I rebased and removed the Android entry, as this made it into the notes for `android-6.4.0-alpha.1`.

**Pull request merged**